etc/scripts/json2csv.py

```diff
@@ -163,7 +163,7 @@ def flatten_scan(scan):
                             pack[nk] = '\n'.join(val)
                         if k == 'asserted_licenses':
                             # All license names are to be joined in a multi-line cell
-                            licenses = [license_info['license'] for license_info in val]
+                            licenses = [license_info.get('license', '') or '' for license_info in val]
```

Comment:
good point! so for simplicity and clarity you could use instead this:
`license_info.get('license') or ''`
e.g. if license is really a `None` or not present, you will get a empty string